changelog.txt

```diff
@@ -1,3 +1,29 @@
+1.9.0-beta3 (2019-02-24)
+------------------------------------------------------------------------
+- Feature: Option to adjust font size separately from GUI size (#7003)
+- Feature: Increase maximum number of orders from 64000 to ~16.7m (#7220)
+- Add: Show performance of AI and GS in framerate window
+- Add: News menu entry and shortcut for deleting all messages (#7240)
+- Change: [OSX] Improved scrolling behaviour when using touchpads
+- Change: Add scrollbar to cargo legend in cargo payment rates window
+- Change: Owner of vehicle with exclusive transport rights may now load cargo from neutral stations (#7256)
+- Change: Improved UI behaviour when dragging sliders with the mouse (#7227)
+- Change: Use selected group as parent when creating a new group (#7224)
```

Comment:
s/group/vehicle group/g ?

Reply:
Yeah, but still, bad commit message

Reply:
That hardly matters, almost all the other changelog lines are adjusted from the actual commit messages